### PR TITLE
Walls and Floors PR 4

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
image
image
image

It's not accurate to wonderlabs... But It'll do.

Floor sprites from TGstation.

## Why It's Good For The Game
Hopefully gives us a new look

## Changelog
:cl:
imageadd: reworks our look.
/:cl:

### CODE REVIEW

File: code/game/turfs/open/floor/misc_floor.dm [hunk 1]
```diff
@@ -9,6 +9,10 @@
 	icon_state = "rockvault"
 	floor_tile = /obj/item/stack/tile/plasteel
 
+/turf/open/floor/tile
+	icon_state = "floor_plates"
+	floor_tile = /obj/item/stack/tile/plasteel
```
Comment (EgorDinamit) [inline code/game/turfs/open/floor/misc_floor.dm:14]: While you are at it, might want to add a proper stack item for this.
It's not necessary, but would be nice if people could replace floors using it, if they find a crowbar.

File: code/game/turfs/open/lobotomy.dm [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/turf/open/floor/facility
```
Comment (EgorDinamit) [inline code/game/turfs/open/lobotomy.dm:1]: Same for all floors there.

**Comment by EgorDinamit** [comment]
Overall, walls look good, although I'd prefer to keep it test-merged for a while to see how others react to it.

File: code/game/turfs/open/lobotomy.dm (hunk 2 above)
Comment (Kitsunemitsu) [reply code/game/turfs/open/lobotomy.dm:1]: Just did what all the other plasteel tiles do and default to the plasteel.

**Comment by Kitsunemitsu** [comment]
> Airlocks open very janky, the animation of icons may need to be checked. Animation with paper needs to be changed to match how the airlocks open vertically and the open airlock sprite could be made more raised.

Fixed the animation , paper isn't done per airlock though.

File: code/game/turfs/open/lobotomy.dm (hunk 2 above)
Comment (EgorDinamit) [reply code/game/turfs/open/lobotomy.dm:1]: That's a bad idea: Removing floor will result in default tiles being dropped.

File: code/game/turfs/open/floor/fancy_floor.dm [hunk 3]
```diff
@@ -18,7 +18,7 @@
 	tiled_dirt = FALSE
 
 /turf/open/floor/wood/setup_broken_states()
-	return list("wood-broken", "wood-broken2", "wood-broken3", "wood-broken4", "wood-broken5", "wood-broken6", "wood-broken7")
+	return list("wood")
```
Comment (EgorDinamit) [inline code/game/turfs/open/floor/fancy_floor.dm:21]: Why?

Comment (EgorDinamit) [reply code/game/turfs/open/floor/fancy_floor.dm:21]: Remove that proc altogether, and what it is used for as well.

Comment (EgorDinamit) [reply code/game/turfs/open/floor/fancy_floor.dm:21]: Actually, forget it.

**Review by EgorDinamit — APPROVED** [review]

**Pull request merged**